Index: lib/util/CMakeLists.txt
===================================================================
--- lib/util/CMakeLists.txt
+++ lib/util/CMakeLists.txt
@@ -53,8 +53,8 @@
   ${GLUT_LIBRARIES} 
   ${QHULL_LIBRARIES}
   GLEW
-  boost_thread-mt
-  boost_system-mt
+  boost_thread
+  boost_system
   )
 
 set(target hrpsysUtil)
Index: lib/util/CMakeLists.txt
===================================================================
--- util/simulator/CMakeLists.txt
+++ util/simulator/CMakeLists.txt
@@ -40,7 +40,7 @@
      )
 else()
    target_link_libraries(hrpsysext 
-     boost_python-mt
+     boost_python
      hrpsysUtil
      ${PYTHON_LIBRARIES}
      )
